pygithub/im_commits_pygithub.py

```diff
@@ -0,0 +1,66 @@
+"""Demo designed to find and store information about PyGithub."""
+from github import Github
+from pydriller import RepositoryMining, GitRepository
+from pydriller.domain.commit import ModificationType
+from fastapi import FastAPI
+
+test_dict = {}
+def output_hash_map(dictionary):
+    """Method to print out the hash map."""
+    # print headings
+    print("Branch", "\t Author", "\t Time", "\t Files", "\t Message")
+    # prints hashmap content
+    # for key in dictionary:
+
+
+    for key, _ in dictionary.items():
+        print(key)
+        for items in dictionary[key]:
+            for item in items:
+                print("\t" + str(item) + "\t")
+            # print("\t" + str(item))
+
+def get_repo_commits_py_github():
+    """Method to get commit information using pygithub"""
+    user_token = input("Enter User Token: ")
+    user = Github(user_token)
+    print(user.get_user().login)
+    username = user.get_user().login
+    repo = user.get_repo("GatorCogitate/cogitate_tool")
+    allcommits = repo.get_commits()
+    all_branches = repo.get_branches()
+
+    branches = {}
+
+    for branch in all_branches:
+        # Create hash map.
+        data_list = list()
```

Comment:
I think this is just a list, not a hash map. Also, to declare an empty list, you can just do `data_list = []`